api_tests/harness.ts

```diff
@@ -59,43 +60,63 @@ async function runTests(testFiles: string[]) {
         logDir
     );
 
-    process.on("SIGINT", () => {
+    async function cleanupAll() {
+        try {
+            btsieveRunner.stopBtsieve();
+            nodeRunner.stopCnds();
+            await ledgerRunner.stopLedgers();
+        } catch (_) {
+            // we don't care much if the cleanup fails
```

Comment:
We can of do if we have some rogue processes/docker runnings. A `console.log` should be enough.